src/gpodder/gtkui/main.py

```diff
@@ -2805,16 +2805,22 @@ def update_feed_cache_finish_callback():
                     # download older episodes first
                     episodes = list(Model.sort_episodes_by_pubdate(episodes))
 
-                if not episodes:
+                # Remove episodes without downloadable content
+                downloadable_episodes = [e for e in episodes if e.url]
+
+                if not downloadable_episodes:
                     # Nothing new here - but inform the user
                     self.pbFeedUpdate.set_fraction(1.0)
-                    self.pbFeedUpdate.set_text(_('No new episodes'))
+                    self.pbFeedUpdate.set_text(
+                        _('No new episodes with downloadable content') if episodes and not downloadable_episodes else _('No new episodes'))
```

Comment:
`and not downloadable_episodes` is redundant here, because of the if-clause above.

Reply:
Oh right, thanks!